### PR TITLE
JBIDE-20190 - add a new repository for fetching some client jars

### PR DESCRIPTION
**Pull request by robstryker**

### CODE REVIEW

**Comment by mickaelistria** [comment]
Does it have to be in both repositories and pluginRepositories ?

File: parent/pom.xml [hunk 1]
```diff
@@ -928,6 +936,14 @@
 				<enabled>true</enabled>
 			</snapshots>
 		</pluginRepository>
+                <pluginRepository>
+                        <id>redhat-maven-repository</id>
```
Comment (maxandersen) [inline parent/pom.xml]: this should not be necessary. we don't want to accidentally pick up maven plugins from this repo if can be avoided.

File: parent/pom.xml [hunk 2]
```diff
@@ -899,6 +899,14 @@
 				<enabled>true</enabled>
 			</snapshots>
 		</repository>
+		<repository>
+			<id>redhat-maven-repository</id>
```
Comment (maxandersen) [inline parent/pom.xml]: use the name "jboss-ga-repository" to be aligned with naming used in official docs: https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.2/html/Development_Guide/Configure_the_JBoss_Enterprise_Application_Platform_Maven_Repository_Using_the_Maven_Settings.html

**Comment by robstryker** [comment]
I have updated the PR, but I'm not sure I have the bandwidth to test it properly.  But then, I see no reason why the updated patch shouldn't work (famous last words, eh?)

**Comment by maxandersen** [comment]
looks right. +1

**Pull request merged**